src/OmniSharp/Services/DocumentationConverter.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+
+namespace OmniSharp.Documentation
+{
+    public static class DocumentationConverter
+    {
+        /// <summary>
+        /// Converts the xml documentation string into a plain text string.
+        /// </summary>
+        public static string ConvertDocumentation(string xmlDocumentation, string lineEnding)
+        {
+            if (string.IsNullOrEmpty(xmlDocumentation))
+                return string.Empty;
+
+            var reader = new StringReader("<docroot>" + xmlDocumentation + "</docroot>");
+            using (var xml = XmlReader.Create(reader))
+            {
+                var ret = new StringBuilder();
+
+                try
+                {
+                    xml.Read();
+                    string elementName = null;
+                    do
+                    {
+                        if (xml.NodeType == XmlNodeType.Element)
+                        {
+                            elementName = xml.Name.ToLowerInvariant();
+                            switch (elementName)
+                            {
+                                case "filterpriority":
+                                    xml.Skip();
+                                    break;
+                                case "remarks":
+                                    ret.Append(lineEnding);
+                                    ret.Append("Remarks:");
+                                    ret.Append(lineEnding);
+                                    break;
+                                case "example":
+                                    ret.Append(lineEnding);
+                                    ret.Append("Example:");
+                                    ret.Append(lineEnding);
+                                    break;
+                                case "exception":
+                                    ret.Append(lineEnding);
+                                    ret.Append(GetCref(xml["cref"]));
```

Comment:
@nosami - we should trim the result from ```GetCref``` because otherwise it reads ```Exception : some text, some text``` (whitesapce before ```:```)